qemu-cc/qemu-cc.spec

```diff
@@ -0,0 +1,214 @@
+Name: qemu-cc
+Version: v2.9.0+git.a2e6957b6f
+Release: 0
+Source0: %{name}-%{version}.tar.gz
+Source1: qemu-cc-rpmlintrc
+BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}
+
+Summary  : OpenBIOS development utilities
+Group    : Development/Tools
+License  : BSD-2-Clause BSD-3-Clause GPL-2.0 GPL-2.0+ LGPL-2.0+ LGPL-2.1
+
+Requires: qemu-cc-bin
+Requires: qemu-cc-data
+
+BuildRequires : automake
+BuildRequires : bison
+BuildRequires : flex
+BuildRequires : glib2-devel
+BuildRequires : libattr-devel
+BuildRequires : libcap-devel
+BuildRequires : libcap-ng-devel
+BuildRequires : libtool
+BuildRequires : libtool-ltdl-devel
+BuildRequires : libtool
+BuildRequires : m4
+BuildRequires : numactl-devel
+BuildRequires : python-devel
+BuildRequires : zlib
+BuildRequires : pkgconfig(pixman-1)
+Patch1: configure.patch
+
+%undefine _missing_build_ids_terminate_build
+%define  debug_package %{nil}
+
+%description
+===========
+QEMU is a generic and open source machine & userspace emulator and
+virtualizer.
+
+%package bin
+Summary: bin components for the qemu-cc package.
+Group: Binaries
+Requires: qemu-cc-data
+
+%description bin
+bin components for the qemu-cc package.
+
+
+%package data
+Summary: data components for the qemu-cc package.
+Group: Data
+
+%description data
+data components for the qemu-cc package.
+
+%package doc
+Summary: docs for the qemu-cc package.
+Group: Data
+
+%description doc
+Docs for the qemu-cc package.
+
+%prep
+%setup -q
+%patch1 -p1
+
+%build
+export LANG=C
+%configure --prefix=/usr \
+	   --datadir=/usr/share/qemu-cc \
+	   --libdir=/usr/lib64/qemu-cc \
+	   --libexecdir=/usr/libexec/qemu-cc \
+	   --disable-tools \
```

Comment:
After some discussion on the public irc channel, we've decided it makes sense to land this first, then at at later date rework all scripts/config files that need qemu options to call the script on https://github.com/clearcontainers/tests/pull/340.

As a mimimum, this PR needs to add `--enable-vhost-net`. The other options listed above either *should* be fine to leave out for now (will not compromise functionality). The issue of the security options probably needs further discussion on https://github.com/clearcontainers/tests/pull/340 as we're not building with those options anywhere afaik.